src/admin-views/aggregator/tabs/import-form.php

```diff
@@ -1,5 +1,6 @@
 <?php
-$has_license_key = Tribe__Events__Aggregator::is_service_active();
+$has_license_key = tribe( 'events-aggregator.main' )->is_service_active();
```

Comment:
I still feel like we _shouldn't_ (this implies it's an instance method and operates on an instance: it doesn't) but if it's the convention we're using elsewhere that's fine